src/usr/share/gpgmailer/mailsender.py

```diff
@@ -97,6 +100,7 @@ def sendmail(self, message_string, recipients):
             self._connect()
 
         try:
+            self.logger.debug(self.config['sender'])
```

Comment:
Again, if this log message is going to remain here, it should have more context.